_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1041,11 +1043,76 @@ private function _process_onboarding( $data ) {
 			}
 		}
 
+		// Add/update business address widget
+		if ( isset( $data['businessAddress'] ) ) {
+			//grab the first sidebar and proceed if it is present
+			$first_sidebar = self::get_first_sidebar();
+
+			if( $first_sidebar ) {
+				$title = wp_unslash( $data[ 'businessAddress' ][ 'name' ] );
```

Comment:
Do we want to store `$data[ 'businessAddress' ]` in a variable to make the code a little shorter by accessing all those fields with less code?